ctru-rs/src/applets/error.rs

```diff
@@ -0,0 +1,56 @@
+//! Error applet
+//!
+//! This applet displays error text as a pop-up message on the lower screen.
+#![doc(alias = "Error")]
```

Comment:
Nit: I don't know if it makes any difference, but we usually write doc aliases in lowercase.
```suggestion
#![doc(alias = "error")]
```

Reply:
Turns out there's no need for an alias since the module is already named `error` anyway